Dockerfile.integration-test

```
@@ -0,0 +1,14 @@
+FROM bats/bats:latest
```

I would like to avoid the latest tag and just use an exact versioning what we update
```suggestion
FROM bats/bats:1.11.0
```

---

I have fixed in https://github.com/cloudoperators/greenhouse-extensions/pull/237/commits/924cf43397727fe509b9c81e936192123c6d484f commit.